conf/modules.config

```diff
@@ -339,11 +339,13 @@ process {
         ext.args          = '--everything --filter_common --per_gene --total_length --offline'
         ext.container_tag = { "104.3.${params.genome}" }
         publishDir        = [ enabled: false ]
+        container         = { "nfcore/vep:${task.ext.container_tag}" }
```

Comment:
```suggestion
        container = { "nfcore/vep:104.3.${params.genome}" }
        publishDir = [ enabled: false ]
```

Reply:
I am not sure yet if this completely correct. i pushed because I need help with the split_fastq tests. The results are nowhere to be found

Reply:
do you have any idea?

Reply:
I'll check your branch